src/pages/sighting/SightingCore.jsx

```diff
@@ -86,9 +90,18 @@ export default function SightingCore({
 
   useDocumentTitle(`Sighting ${id}`, { translateMessage: false });
 
+  const [localDeleteError, setLocalDeleteError] = useState(null);
   const [historyOpen, setHistoryOpen] = useState(false);
   const [deleteDialogOpen, setDeleteDialogOpen] = useState(false);
-
+  useEffect(
+    () => {
+      const localDeleteErrorMessage = pending
+        ? deleteAssetGroupSightingError
+        : deleteSightingError;
+      setLocalDeleteError(localDeleteErrorMessage);
+    },
+    [deleteAssetGroupSightingError, deleteSightingError],
+  );
```

Comment:
why not just `const localDeleteError = pending ? deleteAssetGroupSightingError : deleteSightingError;`. not fully seeing the point of useEffect and useState here